cmake/Reprojucer.cmake

```diff
@@ -1622,33 +1635,51 @@ function(jucer_project_end)
         unset(auv3_target)
       endif()
 
-      if(TARGET ${target}_AUv3_AppExtension)
-        set(standalone_target ${target}_AUv3_Standalone)
-        add_executable(${standalone_target} MACOSX_BUNDLE
+      if(JUCER_BUILD_AUDIOUNIT_V3
+          AND DEFINED JUCER_VERSION AND JUCER_VERSION VERSION_LESS 5)
+        set(juce4_standalone ON)
+      endif()
+
+      if(juce4_standalone OR (JUCER_BUILD_STANDALONE_PLUGIN
+          AND NOT (DEFINED JUCER_VERSION AND JUCER_VERSION VERSION_LESS 5)))
+        if(juce4_standalone)
+          set(standalone_target ${target}_AUv3_Standalone)
+        else()
+          set(standalone_target ${target}_StandalonePlugin)
+        endif()
+        add_executable(${standalone_target} WIN32 MACOSX_BUNDLE
```

Comment:
`WIN32 MACOSX_BUNDLE`? Does that make sense?

Reply:
Yes. It means that this executable is a GUI application. `WIN32` will be translated to the linker flag `/SUBSYSTEM:WINDOWS` on Windows, and `MACOSX_BUNDLE` will produce an `.app` bundle on macOS. These settings don't have any impact on other platforms.